src/StaticFAQ/index.js

```diff
@@ -70,8 +77,14 @@ class StaticFAQ extends React.Component<Props, State> {
             onChange={this.handleSearchChange}
             placeholder="What can we help you with?"
             icon={<Magnify />}
+            withCancel
+            resetInput={this.handleCancelSearch}
```

Comment:
I would rename `resetInput` e.g. to `onReset` so names of all callback properties are consistent